mkdocs.yml

```diff
@@ -193,6 +193,7 @@ nav:
       - Switching Backstage from SQLite to PostgreSQL: 'tutorials/switching-sqlite-postgres.md'
       - Using the Backstage Proxy from Within a Plugin: 'tutorials/using-backstage-proxy-within-plugin.md'
       - Migration to Yarn 3: 'tutorials/yarn-migration.md'
+      - Migration to Material UI v5: 'tutorials/migrate-to-mui5.md'
```

Comment:
This is great but you'll also want to update this file too: https://github.com/backstage/backstage/blob/00946caf9ad08b3103827779b7375763a32a3278/microsite/sidebars.json#L402